.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,42 @@
+name: 🐛 Bug
+description: Create a bug report to help us improve
+labels: ['Bug']
```

Should be lowercase to match our issue label (which I think is case-sensitive).

```suggestion
labels: ['bug']
```